lib/client.js

```diff
@@ -466,90 +477,81 @@ var GiantSwarm = (function () {
       this._simple_data_post_request(uri, successCallback, errorCallback);
     },
 
-
-    /**
-     * Get status of a service
+   /**
+     * Get status of a component
      *
      * @param {String} [organizationName] Name of the organization
      * @param {String} [environmentName] Name of the environment
-     * @param {String} [applicationName] Name of the application
      * @param {String} [serviceName] Name of the service
+     * @param {String} [componentName] Name of the component
      * @param {Function} [successCallback] Callback receiving the status information object as argument
      * @param {Function} [errorCallback] Optional callback to be called in case of an error, error object as argument
      */
-    // serviceStatus: function(organizationName, environmentName, applicationName, serviceName, successCallback, errorCallback) {
-    //   this._validateStringParameters({
-    //     organizationName: organizationName,
-    //     environmentName: environmentName,
-    //     applicationName: applicationName,
-    //     serviceName: serviceName
-    //   });
-    //   var uri = '/v1/org/' + encodeURIComponent(organizationName) + '/env/' + encodeURIComponent(environmentName) + '/app/' + encodeURIComponent(applicationName);
-    //   uri += '/service/' + encodeURIComponent(serviceName) + '/status';
-    //   this._simple_data_get_request(uri, null, successCallback, errorCallback);
-    // },
+    componentStatus: function(organizationName, environmentName, serviceName, componentName, successCallback, errorCallback) {
+      this._validateStringParameters({
+        organizationName: organizationName,
+        environmentName: environmentName,
+        serviceName: serviceName,
+        componentName: componentName
+      });
+
+      params = JSON.stringify({component: componentName})
+
+      var uri = '/v1/org/' + encodeURIComponent(organizationName) + '/env/' + encodeURIComponent(environmentName);
```

Comment:
these lines repeat them self a few times - maybe move to a separate helper function?